libraries/abstractions/ble_hal/include/bt_hal_manager_adapter_ble.h

```diff
@@ -959,6 +959,19 @@ typedef struct
                                                bool bSetScanRsp,
                                                uint8_t * pucData,
                                                size_t xDataLen );
+
+    /**
+     *
+     * @brief Sets the LE background scan interval and window in units of N*0.625 msec.
+     *
+     * @param[in] ucAdapterIf Adapter interface ID. Returned from BTRegisterBleAdapterCallback_t after calling pxRegisterBleApp.
+     * @param[in] ulScanInterval scan interval time in units of 0.625Ms
```

Comment:
Parameter names need to match the ones on lines 973/974.

Reply:
Ops! Fixed that